configure.py

```diff
@@ -53,7 +53,7 @@
 valid_mips_fpu = ('fp32', 'fp64', 'fpxx')
 valid_mips_float_abi = ('soft', 'hard')
 valid_intl_modes = ('none', 'small-icu', 'full-icu', 'system-icu')
-with open ('tools/icu/icu_versions.json') as f:
+with open('tools/icu/icu_versions.json', encoding='utf-8') as f:
```

Comment:
If we are going to modify these, let's use [pathlib](https://docs.python.org/3/library/pathlib.html#pathlib.Path.read_text) and make sure we can pass [`ruff --select=PTH configure.py`](https://beta.ruff.rs/docs).